Lombiq.ChartJs.Samples/ResourceManagementOptionsConfiguration.cs

```diff
@@ -1,19 +1,30 @@
 using Microsoft.Extensions.Options;
+using OrchardCore.Modules.Manifest;
 using OrchardCore.ResourceManagement;
+using System.Linq;
+using System.Reflection;
 
 namespace Lombiq.ChartJs.Samples;
 
 public class ResourceManagementOptionsConfiguration : IConfigureOptions<ResourceManagementOptions>
 {
     private static readonly ResourceManifest _manifest = new();
 
-    static ResourceManagementOptionsConfiguration() =>
+    static ResourceManagementOptionsConfiguration()
+    {
+        var moduleVersion = typeof(ResourceManagementOptionsConfiguration)
+            .Assembly
+            .GetCustomAttributes<ModuleAttribute>()
+            .First()
+            .Version;
+
         _manifest
             .DefineStyle("Lombiq.ChartJs.Samples")
             .SetUrl(
                 "~/Lombiq.ChartJs.Samples/css/chartjs-samples.min.css",
                 "~/Lombiq.ChartJs.Samples/css/chartjs-samples.css")
-            .SetVersion("1.0.0");
+            .SetVersion(moduleVersion);
```

Comment:
Is it necessary to set a version for some reason here? Because this resource is not used from other modules, so having a version doesn't really bring benefits (since that's only useful if you want to allow multiple versions to be referenced).